src/watch/watch.go

```diff
@@ -151,3 +143,16 @@ func addSource(watcher *fsnotify.Watcher, state *core.BuildState, source core.Bu
 		}
 	}
 }
+
+// commands returns the plz commands that should be used for the given labels.
+func commands(state *core.BuildState, labels []core.BuildLabel, run bool) []string {
+	if run {
+		return []string{"run", "parallel"}
```

Comment:
Why do you want to default to parallel?

Reply:
Mostly to allow for it to accept multiple targets. I was tossing up between `parallel` or `sequential` and settled on this - happy to change if you think sequential would make more sense.

Reply:
Does it make sense to allow multiple targets to `run` at all?

Reply:
if someone passes multiple targets, why not? I'm not sure it will be common but we've had sufficient need for that to have added the `sequential` and `parallel` subcommands.

Reply:
Fair enough.